### ISSUE TITLE
Task 4 :Create plots showing accumulated GDD vs time for selected cities.

### ISSUE THREAD

**Issue by chestye** [srp112/GDD_Project#9]
Hey guys, i  have an issue with plotting the cumulative graph . Keep getting with an empty plot in it.Also a dimension error. Anyone know how i can solve these issues.

**Comment by uzipaz** [comment]
I have fixed the issues you described, its been updated. The program will now draw and save plots as image files into 'plots' folder.

**Comment by chestye** [comment]
thanks. Will check when i get home

**Comment by liangyeni123** [comment]
From the sample picture professor provided, in my opinion, since the GDD plot are based on accumulated value, there won't be a drop after the peak in the curve, which means it's should grow bigger and bigger...

**Comment by uzipaz** [comment]
I agree with you. So far, the program only plots GDD values as they are. They are not accumulated. As of now, I am not sure how the accumulated GDD plot can be implemented or how accumulated GDD can be calculated? if you have the knowledge about this, then feel free to contribute to the program, everything is in place, all you have to do add the math in it.

**Comment by liangyeni123** [comment]
I tried to solve this problem but i am not sure how accumulated GDD can be calculated too. Anyone else could solve it?

**Comment by chestye** [comment]
I also tried alot to do it,but wasn't
I was thinking of asking the TA tomorrow for any ideas since the Professor
won't be around.

**Comment by fmkazemi** [comment]
If i'm not mistaken we have to use cumsum()  function for solving this problem.
